### DISCUSSION TITLE
Gmsh to Neper

### DISCUSSION THREAD

**Discussion by inandi2** [neperfepx/neper#731] [Q&A] [answered]
Hello all,
Is there any way to introduce the polycrystalline feature to a geometry created using Gmsh?

Thank you,
Indrajit

**Comment by rquey** [comment, marked as answer]
It is possible to export Neper tessellations into a Gmsh geo file using `-format geo`, and the Neper meshes are written in a Gmsh-compliant `msh` format.  It should be possible to use these files in Gmsh.
